From 033c0c2c8b45c3f1da0a9c82a09a5121a4c027dc Mon Sep 17 00:00:00 2001
From: Fathi Boudra <fathi.boudra@linaro.org>
Date: Mon, 20 Mar 2017 11:09:20 +0200
Subject: [PATCH 2/3] selftests: ftrace: add config fragment

ftrace/kprobe tests require kprobes events. Enable kprobes to run these
tests.

Signed-off-by: Fathi Boudra <fathi.boudra@linaro.org>
---
 tools/testing/selftests/ftrace/config | 2 ++
 1 file changed, 2 insertions(+)
 create mode 100644 tools/testing/selftests/ftrace/config

diff --git a/tools/testing/selftests/ftrace/config b/tools/testing/selftests/ftrace/config
new file mode 100644
index 000000000000..8a1c9f949fe0
--- /dev/null
+++ b/tools/testing/selftests/ftrace/config
@@ -0,0 +1,2 @@
+CONFIG_KPROBES=y
+CONFIG_FTRACE=y
-- 
2.11.0

